rpcs3/rpcs3qt/settings_dialog.ui

```diff
@@ -36,7 +36,7 @@
       </sizepolicy>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>8</number>
```

Comment:
why is everyone commit this LMAO

Reply:
idk, its something the editor changes automatically or something.

Reply:
I know. just weird to commit it everytime.
It's similar to changing the default value of an unused bool member variable everytime you commit a file lol.